main.go

```diff
@@ -44,8 +44,10 @@ func init() {
 }
 
 func main() {
+	var namespace string
 	var metricsAddr string
 	var enableLeaderElection bool
+	flag.StringVar(&namespace, "namespace", "kubeflow", "Namespace within which CRD controller is running.")
```

Comment:
please remove kubeflow. Let the default be ""

Reply:
done